packages/renderer/src/lib/kube/pods/PodDetails.spec.ts

```diff
@@ -0,0 +1,137 @@
+/**********************************************************************
+ * Copyright (C) 2023-2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import type { CoreV1Event, KubernetesObject, V1Pod } from '@kubernetes/client-node';
+import { fireEvent, render, screen, waitFor } from '@testing-library/svelte';
+import { writable } from 'svelte/store';
+import { router } from 'tinro';
+import { beforeAll, beforeEach, expect, test, vi } from 'vitest';
+
+import { lastPage } from '/@/stores/breadcrumb';
+import * as kubeContextStore from '/@/stores/kubernetes-contexts-state';
+
+import PodDetails from './PodDetails.svelte';
+
+const mocks = vi.hoisted(() => ({
+  TerminalMock: vi.fn(),
+}));
+vi.mock('@xterm/xterm', () => ({
+  Terminal: mocks.TerminalMock,
+}));
```

Comment:
```suggestion
vi.mock('@xterm/xterm');
```

This is not needed, you should be able to access terminal with `expect(Terminal)` or `vi.mocked(Terminal)`.

Reply:
Fixed in new commit.